mods/ra2/weapons/zaps.yaml

```diff
@@ -230,3 +230,45 @@ CRNeutronRifle:
 	Range: 6c0
 	Warhead@Disable: GrantExternalCondition
 		Duration: 350
+
+LightningStorm:
+	ReloadDelay: 0
+	Range: 7c0
+	Projectile: InstantHit
+		Blockable: false
+	Warhead@1Eff: CreateEffect
+		Delay: 0
+		Explosions: weathercloud1, weathercloud2, weathercloud3
+		UsePlayerPalette: true
+		ExplosionPalette: player
+		ValidTargets: Ground, Water
+		ImpactActors: false
+	Warhead@2Dam: SpreadDamage
+		Delay: 56
+		Spread: 3c0
+		Damage: 165
+		Versus:
+			None: 0
+			Flak: 0
+			Plate: 0
+			Light: 0
+			Medium: 0
+			Heavy: 0
+			Wood: 100
+			Steel: 100
+			Concrete: 6
+			Drone: 0
+			Rocket: 0
```

Comment:
Those values are okay? Those 0 there

Reply:
Changed the values in this part, and it started to affect units.

Reply:
btw the OG ra2 yuris revenge verses are:

```
; Ion storm strike ;lightning strike
[IonWH]
Wall=yes
Wood=yes
CellSpread=2.0
PercentAtMax=.5
Verses=100%,100%,100%,100%,100%,100%,100%,100%,2%,100%,100%
;;Verses=100%,100%,100%,100%,100%,100%,100%,100%,3%,100%,100%
Conventional=yes
Rocker=yes
InfDeath=5
;AnimList=XGRYSML1,XGRYSML2,EXPLOSML,XGRYMED1,XGRYMED2,EXPLOMED,EXPLOLRG
AnimList=EXPLOSML
;Deform=10%
;DeformThreshhold=300
;Tiberium=yes
;Sparky=no
Bright=yes
CombatLightSize=40%	; SJM: Force size of combat light, overriding damage-based value
```

Reply:
Made it more deadly.